src/plugins/index_pattern_management/public/plugin.ts

```diff
@@ -109,7 +121,52 @@ export class IndexPatternManagementPlugin
       },
     });
 
-    return this.indexPatternManagementService.setup({ httpClient: core.http });
+    const registerLibrarySubApp = () => {
+      core.application.register({
+        id: IPM_APP_ID,
+        title: sectionsHeader,
+        order: 8100,
+        category: DEFAULT_APP_CATEGORIES.opensearchDashboards,
+        mount: async (params: AppMountParameters) => {
+          const { mountManagementSection } = await import('./management_app');
+
+          const [coreStart] = await core.getStartServices();
+
+          const setBreadcrumbsScope = (
+            crumbs: ChromeBreadcrumb[] = [],
+            appHistory?: ScopedHistory
+          ) => {
+            const wrapBreadcrumb = (item: ChromeBreadcrumb, scopedHistory: ScopedHistory) => ({
+              ...item,
+              ...(item.href ? reactRouterNavigate(scopedHistory, item.href) : {}),
+            });
+
+            coreStart.chrome.setBreadcrumbs([
+              ...crumbs.map((item) => wrapBreadcrumb(item, appHistory || params.history)),
+            ]);
+          };
+
+          const managementParams: ManagementAppMountParams = {
+            element: params.element,
+            history: params.history,
+            setBreadcrumbs: setBreadcrumbsScope,
+            basePath: params.appBasePath,
```

Comment:
Do we need to use basePath here?

Reply:
basePath is not used, but it's not optional in `ManagementAppMountParams`, so i put the value here.